config-templates/ibm/ocs/4.6_local/custom-parameters.json

```diff
@@ -0,0 +1,63 @@
+[
+  {
+    "description": "Cluster Name of OCS",
+    "displayname": "Cluster Name",
+    "name": "ocs-cluster-name",
+    "required": "true"
+  },
+  {
+    "description": "Enter the OSD device path. Mandatory",
+    "displayname": "Osd Device Path",
+    "name": "osd-device-path",
+    "required": "true",
+    "type": "csv"
+  },
+  {
+    "description": "Enter the MON device path. Mandatory",
+    "displayname": "Mon device Path",
+    "name": "mon-device-path",
+    "required": "true",
+    "type": "csv"
+  },
+  {
+    "description": "Enter the number of OSD. Optional. Default is 1",
+    "displayname": "Num of osd",
+    "name": "num-of-osd",
+    "default": "1",
+    "required": "false"
+  },
+  {
+    "description": "Worker Nodes you want to deploy OCS on. Min 3. Leave if you want OCS on all workers",
+    "displayname": "worker nodes",
+    "name": "worker-nodes",
+    "required": "false",
+    "type": "csv"
+  },
+  {
+    "description": "OCS Upgrade",
+    "displayname": "OCS upgrade",
+    "name": "ocs-upgrade",
+    "default": "false",
+    "required": "false",
+    "type": "boolean"
+  },
+  {
+    "description": "Billing type",
+    "displayname": "Billing type",
+    "name": "billing-type",
+    "default": "hourly",
+    "required": "false"
+  },
+  {
+    "description": "IBM COS access key",
+    "displayname": "IBM COS access key",
+    "name": "ibm-cos-access-key",
+    "required": "false"
+  },
+  {
+    "description": "IBM COS secret key",
+    "displayname": "IBM COS secret key",
```

Comment:
It looks secret, Can we add  category="secret" ?

Reply:
@prgavali For `category="secret"` the parameter value is getting encoded to base64
@Shirisha-S-Rao-1 Is the resource expecting base64 or plain text?